docs/consensus/consensus/crypto_primitives.md

```diff
@@ -29,7 +29,7 @@ We currently use Schnorr/Ristretto x25519 (also known as sr25519) as the key der
 ## Erasure Code
 
 An erasure code extends the given data so that the original data can be recovered from a subset and protected against loss.
-In Subspace, erasure code is used to encode and decode blockchain history pieces and their KZG commitments in an archived segment. Erasure coding allows for distributed storage of pieces across farmers and helps protect the data against loss in the event of any failures and network partitions. Erasure code is also used in plotting together with proofs-of-space to create unique, easily recoverable plot files for each farmer.
```

Comment:
Let's keep the following entries in this file untouched, as they are agnostic of the implementation.